src/Types/PlutusData.purs

```diff
@@ -34,6 +35,9 @@ derive instance Eq PlutusData
 derive instance Ord PlutusData
 derive instance Generic PlutusData _
 
+instance DecodeJson PlutusData where
```

Comment:
I don't think we can use a `DecodeJson` instance for `PlutusData`, can we?

Reply:
Will use aeson once it's merged